go-controller/pkg/ovn/master.go

```diff
@@ -996,63 +991,46 @@ func (oc *Controller) syncNodes(nodes []interface{}) {
 
 	for nodeName, chassisName := range chassisMap {
 		if chassisName != "" {
-			_, stderr, err = util.RunOVNSbctl("--if-exist", "chassis-del", chassisName)
+			cmd, err := oc.ovnSBClient.ChassisDel(chassisName)
 			if err != nil {
-				klog.Errorf("Failed to delete chassis with name %s for logical switch %s: stderr: %q, error: %v",
-					chassisName, nodeName, stderr, err)
+				klog.Errorf("Unable to create the ChassisDel command for chassis: %s from the sbdb", chassisName)
+			} else if err = oc.ovnSBClient.Execute(cmd); err != nil {
```

Comment:
same comment as in syncNodesPeriodic to batch execution of the chassisDel commands.